src/pages/Lend/Filter/SelectedRefinements.vue

```diff
@@ -1,38 +1,45 @@
 <template>
 	<div class="selected-refinements">
-		<ais-current-refinements
-			:transform-items="transformItems"
-		>
+		<ais-current-refinements :transform-items="transformItems">
 			<div slot-scope="{ items }">
-				<div class="filter-summary-container">
-					<div>{{ items.length }} Filters Applied</div>
-					<div v-if="isCollapsible" class="show-toggle-container">
-						<div v-if="isCollapsed" @click="handleClickShowMore">Show</div>
-						<div v-else @click="handleClickShowFewer">Hide</div>
+				<div class="row align-middle">
+					<div class="small-9 medium-10 columns">
+						<div :class="[{ collapsed: isCollapsed }, 'accordion-container']">
+							<div class="accordion-content" ref="accordionContent">
+								<filter-chip
+									v-for="item in items"
+									:key="[
+										item.attribute,
+										item.type,
+										item.label,
+										item.operator,
+									].join(':')"
+									:title="`${item.label}`"
+									@click-chip="handleRemoveRefinement(item)"
+								/>
+							</div>
+						</div>
+					</div>
+					<div class="small-3 medium-2 columns">
+						<div class="filter-summary-container">
```

Comment:
The "Show all #" link will need some padding to match the chip height and text alignment of the items on the top row when in the collapsed state.